java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -201,11 +201,15 @@
     private SaltActionChainGeneratorService saltActionChainGeneratorService =
             SaltActionChainGeneratorService.INSTANCE;
 
-    private SaltService saltService = SaltService.INSTANCE;
+    private SystemQuery saltService;
```

Comment:
i left it out to get the more important stuff done first. In the end it should have a name in line with the interface name which means it may actually change again or multiple times when spiting further or deciding on final names interface naming. I will change it for now anyway.